### PR TITLE
Fix crash/panic when in wrong directory

### PR DESCRIPTION
**Pull request by GeertJohan**
The filepath.Walk error should be checked.

### CODE REVIEW

File: app/app.go [hunk 1]
```diff
@@ -19,6 +19,10 @@ func Handle(repoList map[string]string, projectTypeToCheck string, filter string
 			output_channel <- output.Print(fmt.Sprintf("\nChecking '%s' (%s) repos...", projectType, path))
 
 			visit := func(visitedPath string, info os.FileInfo, err error) error {
+				if err != nil {
+					fmt.Printf("error: %v\n", err)
```
Comment (stevenjack) [inline app/app.go]: Theres already a channel for outputting errors, could you change it to something like this:

https://github.com/stevenjack/cig/blob/master/app/app.go#L32

That'll automatically output the error and exit with a non zero.

Thanks for the PR!

**Comment by stevenjack** [comment]
This should resolve https://github.com/stevenjack/cig/issues/26

**Comment by GeertJohan** [comment]
Pushed new version that uses the output channel.

File: app/app.go [hunk 2]
```diff
@@ -19,6 +19,10 @@ func Handle(repoList map[string]string, projectTypeToCheck string, filter string
 			output_channel <- output.Print(fmt.Sprintf("\nChecking '%s' (%s) repos...", projectType, path))
 
 			visit := func(visitedPath string, info os.FileInfo, err error) error {
+				if err != nil {
+					output_channel <- output.Error(err.Error())
```
Comment (stevenjack) [inline app/app.go]: I've merged an update that allows an error to be output without exiting the whole application, as if you have a few folders in your `~/.cig.yaml` and only one of them doesn't exist you don't want this to stop the execution of the application.

With that in mind could you just tweak the output to:

``` go
output_channel <- output.Error(fmt.Sprintf("- %s", err.Error()))
```

so it matches the output when a directory is being listed.

Comment (GeertJohan) [inline app/app.go]: Sure, just a sec.

**Comment by GeertJohan** [comment]
That looks better indeed, although it seems to stop listing packages for a repo's group when an error occured, it continues with the next repo group..

**Comment by GeertJohan** [comment]
Looks better indeed!

**Pull request closed without merging**